src/routes/targeted-messaging/targeted-messaging.controller.spec.ts

```diff
@@ -318,6 +318,79 @@ describe('TargetedMessagingController', () => {
         });
     });
 
+    it('should create a submission for a campaign without targeting enabled', async () => {
+      const outreachId = faker.number.int();
+      const chain = chainBuilder().build();
+      const signerAddress = getAddress(faker.finance.ethereumAddress());
+      const safeAddress = getAddress(faker.finance.ethereumAddress());
+      const submission = submissionBuilder().build();
+      const safe = safeBuilder()
+        .with('owners', [
+          getAddress(faker.finance.ethereumAddress()),
+          signerAddress,
+        ])
+        .build();
```

Comment:
Should `safe.address` be `safeAddress` here?

Reply:
true, no need for both